components/frontend/src/report/ReportSources.test.jsx

```diff
@@ -180,7 +180,7 @@ it("changes the value of a parameter of a source without parameter layout", asyn
         },
         ["source_uuid:0"],
     )
-    await userEvent.type(screen.getAllByLabelText(/URL/)[0], "https://source.org/new{Enter}")
+    await userEvent.type(screen.getAllByLabelText(/URL/)[0], "/new{Enter}")
```

Comment:
I don't understand how this test now covers the `.parameters` addition, do we actually test both the case where they are and are not present?